commerce-cloud-code/app_yourshophere/cartridge/experience/pages/checkoutPage.js

```diff
@@ -0,0 +1,48 @@
+const Template = require('dw/util/Template');
+const HashMap = require('dw/util/HashMap');
+const PageRenderHelper = require('*/cartridge/experience/utilities/PageRenderHelper.js');
+const RegionModelRegistry = require('*/cartridge/experience/utilities/RegionModelRegistry.js');
+
+/**
+ * Render logic for the storepage.
+ *
+ * @param {dw.experience.PageScriptContext} context The page script context object.
+ *
+ * @returns {string} The template text
+ */
+exports.render = function render(context) {
+    try {
+        return renderComponent(context);
```

Comment:
Why not `renderPartial()` here?

Reply:
Because it's a page and no partial

Reply:
But is there an actual difference?

Reply:
not really, but the name. 

I thought about it when writing this page and I like the idea to force the concept that pages are completely defined in the page designer folder to discourage devs circumventing PD